pages/prompt-input/permutations.page.tsx

```diff
@@ -73,6 +73,13 @@ const permutations = createPermutations<PromptInputProps>([
     disableSecondaryActionsPaddings: [false, true],
     disableSecondaryContentPaddings: [false, true],
   },
+  {
+    value: ['Short value for custom primary actions'],
+    actionButtonIconName: [undefined, 'send'],
+    customPrimaryAction: [undefined, 'Custom primary'],
```

Comment:
Nit: I would render an actual button here, not text.

<img width="124" height="231" alt="Image" src="https://github.com/user-attachments/assets/bd1570f1-8c6d-44e0-8a20-cbcd229579ac" />